torch/_dynamo/variables/builtin.py

```diff
@@ -1109,10 +1109,31 @@ def call_setattr(
                 f"setattr(UserDefinedObjectVariable) {type(obj.value).__setattr__}"
             )
         elif isinstance(obj, variables.NNModuleVariable):
+            if isinstance(name_var, ConstantVariable) and obj.is_buffer(tx, name_var):
```

Comment:
What if the module has a custom setattr?

Reply:
Thats ok. We will not raise assertion in export. In eager, we will return that as an output, and then do the mutation in the bytecode.

Reply:
Where did my comment go? Weird. 

Anyway - I'll rewrite it.

If we have a custom setattr, this will not invoke it - as you are simulating the setattr call with an op. Which means that now user code wont get invoked as expected.

If you setattr, youll invoke arbitrary user code, right?